McEngine/src/App/UIFrameworkTest/UIFrameworkTest.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * UIFrameworkTest.cpp
+ *
+ *  Created on: May 28, 2017
+ *      Author: Psy
+ */
+
+#include <UIFrameworkTest/UIFrameworkTest.h>

```

Comment:
Don't use relative includes. (Was probably automatically generated by Eclipse and you didn't change it.)
#include "UIFrameworkTest.h"